### DISCUSSION TITLE
Adding Modals

### DISCUSSION THREAD

**Discussion by HamzaR21** [tailwindlabs/tailwindcss#1687] [General]
Hello there , i was just wondering if there going to be a support for modals anytime soon ?!

**Comment by adamwathan** [comment]
Hey! We already “support” modals in the same sense that we support buttons, cards, drop downs, etc., you are meant to build them yourself using Tailwind’s utility classes.

Here’s an example of a modal built with Tailwind:

https://tailwindui.com/components/application-ui/overlays/modals#component-47a5888a08838ad98779d50878d359b3

If you’re looking for prebuilt components like that, our Tailwind UI project might be what you need 👍🏻

**Reply by HamzaR21** [reply]
Yeah , my bad i didn't see them in the docs , thanks a lot <3
